### PR TITLE
(feat) Tweak aspects of the patient lists workspace

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

These tweaks include:

- Amending the list details view empty state's background color and spacing to match design conventions
- Making the workspace 'closable' (by setting the workspace's `canHide` prop to false). Having a close button is useful because this workspace doesn't have action buttons at the bottom of the screen.
- Setting the workspace `type` of the summary and detail views to the `patient-lists` so that both workspaces can take advantage of the built-in side rail / bottom nav highlighting functionality for active workspaces.

## Screenshots

CleanShot 2023-11-23 at 11  51 02@2x

## Related Issue
*None*

## Other
*None*

### CODE REVIEW

File: packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx [hunk 1]
```diff
@@ -26,6 +26,8 @@ export const SiderailNavButton: React.FC<SiderailNavButtonProps> = ({
 }) => {
   const layout = useLayoutType();
   const { workspaces, workspaceWindowState } = useWorkspaces();
+  console.log('workspaces: ', workspaces);
+  console.log('workspaceWindowState: ', workspaceWindowState);
```
Comment (jayasanka-sack) [inline packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx:30]:
```suggestion
```

Comment (denniskigen) [reply packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx:30]: Yikes, eslint should have caught that.

Comment (jayasanka-sack) [reply packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx]: I know, right? Do you have an idea why it didn't?

**Review by jayasanka-sack — APPROVED** [review]
Thanks, @denniskigen!

File: packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx (hunk 1 above)
Comment (denniskigen) [reply packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx]: Yep, looks like that rule wasn't enabled. Working on a PR to fix that now.

**Review by njiddasalifu — COMMENTED** [review]
LGTM

**Pull request merged**